README.md

```diff
@@ -116,7 +144,10 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+// $( "#greeting " ).click(function() {
+//   $( "body" ).append( $p, [ "hello" ] );
```

Comment:
You're on the right track, but I don't know where `$p` came from and why "hello" is in brackets. `.append` is nice because you can put plain old HTML inside it:

```js
$("body").append("<p>hello</p>");
```